CMakeLists.txt

```diff
@@ -7,9 +7,7 @@ include(GNUInstallDirs)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-
-find_package(USD REQUIRED)
+find_package(USD REQUIRED NAMES pxr)
 find_package(Python COMPONENTS Interpreter Development REQUIRED)
```

Comment:
If I understand well, now Python is not required anymore, so we can remove it ?

Reply:
Indeed, I've removed it.